translate/var_declare/js_var_declare.py

```diff
@@ -0,0 +1,19 @@
+import sys
+import os
+
+# get path automatically and handle imports
+path = os.path.dirname(os.getcwd()) + '/helpers'
+sys.path.insert(1, path)
+import steph_helpers
```

Comment:
probably should just make steph_helpers into js_helpers and then just have a separate py_helpers folder